server/vcr-server/api/v3/views/rest.py

```diff
@@ -246,7 +246,8 @@ def post_verify(self, request, credential_id, presentation_exchange_id):
                 result["result"]["presentaiton"] = presentation_state["presentation"]
```

Comment:
Unrelated to your changes, but `presentaiton` looks like it's a typo that could be making things not work.

Reply:
Should probably log it as a separate issue.

Reply:
https://github.com/bcgov/aries-vcr/pull/604